pysiaf/tests/test_match_v2v3.py

```diff
@@ -1,29 +1,25 @@
 from pysiaf import iando
 from pysiaf.utils import tools
 
+def testit(instrument='NIRCam', apName_1 = 'NRCA1_FULL', apName_2 = 'NRCA5_FULL', verbose=False):
```

Comment:
test function names should start with `test_` (incl. underscore) and be descriptive in what is being tested.